edb/lib/std/30-datetimefuncs.eql

```diff
@@ -28,9 +28,9 @@ std::current_date() -> std::date
 
 
 CREATE FUNCTION
-std::current_time() -> std::time
+std::local_time() -> std::naivetime
```

Comment:
I propose to change the order to `time_local()`; I'd like all time-related functions to be grouped together.

Reply:
We have:
- `current_date`
- `current_datetime`
- `local_time`

Flipping the order seems a bit counter-intuitive.

Reply:
I vote for `current_localtime()`